app/services/duty_options/additional_duty/excise.rb

```diff
@@ -0,0 +1,26 @@
+module DutyOptions
+  module AdditionalDuty
+    class Excise < DutyOptions::AdditionalDuty::Base
+      PRIORIY = 6
+
+      include CommodityHelper
+
+      def duty_calculation_row
+        presented_row = []
+        presented_row << I18n.t(
+          'duty_calculations.options.excise_duty_html',
+          additional_code_description: excise_description,
+        ).html_safe
+
+        presented_row.concat(duty_evaluation.slice(:calculation, :formatted_value).values)
```

Comment:
Again really minor but I think `#values_at` does what you want here ?

Reply:
Nice